Leron Reznikov Election Lab

reznikovl opened the pull request:
Hi Prof K,
This is my lab involving an analysis of the 2016 election. Let me know if it's good or if I should add anything else - thank you!

reznikovl committed:
lr - initial dataset test commit

reznikovl committed:
rough draft of election lab

ykharitonova commented:
Thank you, @reznikovl ! Looks really interesting. I also appreciate it that you took the time to make the lab using the `datascience` module.

- [ ]  Can you please add the source and the URL of the data?

- [ ]  For the column explanations before the `original_data.labels` cell, you might want to format them into a bulleted list for the ease of reading.

- [ ]  For the "adjusted/calculated percentage of people": what are they adjusting for?

- [ ]  Do you know who and how assigns the rating/grade to the pollsters? If you have this information, it would be helpful to include it in the notebook.

- [ ]  To make it easier to reference the different sections of the notebook, could you add **Question N** at the start? E.g. 

> **Question 1.3.** To further understand this relationship, create an array ...

reznikovl committed:
lr - added clarifying points, additional research possibilities, and cleaned formmating

reznikovl commented:
Thank you for the helpful comments! I've incorporated those points into my notebook, and added additional resources for research as well. Let me know if this is good!

(Edit:  I tried to assign you to the PR, but I was only successful in removing myself from it - I think I might not have permission to do that. Sorry if that wasn't the right thing to do!)
